sdk/core/AzureCore/Source/Util/ConvertingInitializers.swift

```diff
@@ -47,54 +47,8 @@ extension URL {
     }
 }
 
-extension Date {
```

Comment:
These have been relocated and consolidated to an `AzureDateFormat` enum that accepts `custom(String)`.